HOWTORELEASE.rst

```diff
@@ -1,7 +1,41 @@
-Here are the steps on how to make a new release.
+=======================
+Releasing pytest-flask
+=======================
 
-1. Create a ``release-VERSION`` branch from ``upstream/master``.
-2. Update ``docs/changelog.rst``.
-3. Push a branch with the changes.
-4. Once all builds pass, push a tag to ``upstream``.
-5. Merge the PR.
+This document describes the steps to make a new ``pytest-flask`` release.
+
+Version
+-------
+
+``master`` should always be green and a potential release candidate. ``pytest-flask`` follows
+semantic versioning, so given that the current version is ``X.Y.Z``, to find the next version number
+one needs to look at the ``docs/changelog.rst`` file.
+
+Steps
+-----
+
+#. Create a new branch named ``release-X.Y.Z`` from the latest ``master``.
+
+#. Create and activate a virtualenv::
```

Comment:
Not sure we need to run a tox command that won't do anything... can you elaborate why the previous instructions were not adequate?

I think the instructions to tag, however, are an improvement and could be kept.

What do you think?

Reply:
> ...can you elaborate why the previous instructions were not adequate?
Nothing wrong with them, the idea was just to break each step down in more detail.

> Not sure we need to run a tox command that won't do anything
We don't. I forgot to convert this to a draft. I meant to provide a description/justification for the changes and clean up the file before opening for review, sorry about that.